build/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@fluentui/internal-tooling",
+  "version": "0.42.0",
+  "license": "MIT",
+  "dependencies": {
+    "@babel/core": "^7.6.4",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.6.2",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/preset-react": "^7.6.3",
+    "@babel/preset-typescript": "^7.6.0",
+    "@babel/register": "^7.4.4",
+    "@babel/standalone": "^7.3.2",
+    "@fluentui/eslint-plugin": "^0.42.0",
+    "@mdx-js/loader": "^1.0.21",
+    "@types/babel__traverse": "^7.0.4",
+    "@types/express": "^4.16.1",
+    "@types/gulp": "^4.0.6",
+    "@types/gulp-babel": "6.1.29",
+    "@types/gulp-cache": "^0.4.4",
+    "@types/gulp-util": "^3.0.34",
+    "@types/gulp-remember": "^0.0.31",
+    "@types/jest-axe": "^2.2.3",
+    "@types/jest": "^24.0.19",
+    "@types/node": "^10.3.2",
+    "@types/puppeteer": "^1.11.1",
+    "@types/rimraf": "^2.0.2",
+    "@types/webpack-dev-middleware": "^2.0.1",
+    "@types/webpack-hot-middleware": "^2.16.3",
+    "@typescript-eslint/eslint-plugin": "^2.5.0",
+    "@typescript-eslint/parser": "^2.5.0",
+    "babel-jest": "^24.5.0",
+    "babel-loader": "^8.0.6",
+    "babel-plugin-lodash": "^3.3.4",
+    "circular-dependency-plugin": "^5.0.2",
+    "clean-webpack-plugin": "^0.1.19",
+    "connect-history-api-fallback": "^1.3.0",
+    "copy-webpack-plugin": "^4.5.2",
+    "del": "^5.1.0",
+    "doctrine": "^2.0.0",
+    "eslint": "^6.5.1",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jest": "^22.4.1",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-react-hooks": "^2.1.2",
+    "eslint-plugin-react": "^7.16.0",
+    "express": "^4.15.4",
+    "extract-comments": "^1.0.0",
+    "fork-ts-checker-webpack-plugin": "^1.3.3",
+    "glob": "^7.1.2",
+    "gulp": "^4.0.2",
+    "gulp-babel": "^8.0.0",
+    "gulp-cache": "^1.0.2",
+    "gulp-remember": "^1.0.1",
+    "gulp-typescript": "^5.0.0",
+    "gulp-util": "^3.0.8",
+    "html-webpack-plugin": "^4.0.0-beta.5",
+    "jest": "^24.9.0",
+    "jest-axe": "^3.1.1",
+    "jest-react-fela": "^10.6.1",
+    "json-stable-stringify-without-jsonify": "^1.0.1",
+    "license-webpack-plugin": "^2.1.1",
+    "portfinder": "^1.0.20",
+    "prettier": "^1.19.1",
+    "progress": "^2.0.3",
+    "puppeteer": "^1.11.0",
+    "react-hot-loader": "^4.12.15",
+    "react-vis": "^1.11.6",
```

Comment:
`reacy-hot-loader` and `react-vis` are docs dependencies

Reply:
Yup, I think I'm going to split the docs tooling out in a different PR (I already have some of the work on that done, but it would make this change larger)